install-dev/upgrade/php/ps_1770_update_tabs.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+/**
+ * File copied from ps_1750_update_module_tabs.php and modified to add new roles
+ */
+function ps_1770_update_tabs()
+{
+    include_once 'add_new_tab.php';
+    include_once 'copy_tab_rights.php';
+
+    add_new_tab_17('AdminParentMailTheme', 'en:Email Themes', 0, false, 'AdminParentThemes');
+    Db::getInstance()->execute(
+        'UPDATE `' . _DB_PREFIX_ . 'tab` SET `active`= 1, `enabled`= 1, `position`= 2 WHERE `class_name` = "AdminParentMailTheme"'
+    );
+
+    // Move AdminMailTheme's parent from AdminMailThemeParent to AdminParentMailTheme
+    $toParentTabId = Db::getInstance()->getValue(
```

Comment:
Ok I get it, you move children tabs from the former AdminMailThemeParent to AdminParentMailTheme
But if some modules already used AdminMailThemeParent as their parent tab then it will become messy, so we'd better keep the original ones and just fix the permissions